lightning/src/ln/msgs.rs

```diff
@@ -1580,6 +1580,9 @@ pub trait OnionMessageHandler {
 	/// drop and refuse to forward onion messages to this peer.
 	fn peer_disconnected(&self, their_node_id: &PublicKey);
 
+	/// Performs actions that should happen on startup and roughly every ten seconds thereafter.
```

Comment:
FWIW, docs tend to be more extensive on this method elsewhere. You may have been waiting for a concept ACK first though.